src/NHSD.BuyingCatalogue.Database/PostDeployment/ProdLikeData/MergeSuppliers.sql

```diff
@@ -0,0 +1,183 @@
+INSERT #Supplier ([Id], [Name], [LegalName], [Summary], [SupplierUrl], [Address], [OdsCode], [CrmRef], [Deleted], [LastUpdated], [LastUpdatedBy]) VALUES (N'10072', N'Push Dr Limited', N'Push Dr Limited', NULL, NULL, N'{"line1":"Arkwright House","line2":"Parsonage","town":"Manchester","county":"Cheshire","postcode":"M3 2LF","country":"England"}', NULL, NULL, 0, CAST(N'2019-08-02T05:22:07.0000000' AS DateTime2), N'00000000-0000-0000-0000-000000000000')
+INSERT #Supplier ([Id], [Name], [LegalName], [Summary], [SupplierUrl], [Address], [OdsCode], [CrmRef], [Deleted], [LastUpdated], [LastUpdatedBy]) VALUES (N'10073', N'iPLATO', N'iPlato Healthcare Ltd', N'iPLATO has been operating mobile patient services in primary care since 2005. In this time we have launched the iPLATO service with over 1,800 GP practices in 75 CCG/PCTs reaching over 25 million patients.
+
+iPLATO is the digital bridge between any of the principal GP IT systems used by the GP network and their patients using one of the most ubiquitous of modern tools – their own mobile phone.
+
+•	Over 90% of adults in Britain own (at least one) mobile phone.
+•	Personal, private and immediate – ideal for health communication.
+•	Preferred by many patients, particularly the ‘hard-to-reach’.
+
+More than 25 million patients in the UK already receive messages from their GP or Healthcare provider through iPLATO Patient Care Messaging for Primary Care system.', N'', N'{"line1":"1 King Street","town":"London","county":"Greater London","postcode":"W6 9HR","country":"England"}', NULL, NULL, 0, CAST(N'2020-04-01T13:48:54.1466667' AS DateTime2), N'00000000-0000-0000-0000-000000000000')
+
+MERGE INTO [dbo].[Supplier] AS TARGET
+USING #Supplier AS SOURCE
+ON TARGET.[Id] = SOURCE.[Id] 
+WHEN MATCHED THEN  
+    UPDATE SET TARGET.[Name] = SOURCE.[Name],
+                TARGET.[LegalName] = SOURCE.[LegalName],
+                TARGET.[Summary] = SOURCE.[Summary],
+                TARGET.[SupplierUrl] = SOURCE.[SupplierUrl],
+                TARGET.[Address] = SOURCE.[Address],
+                TARGET.[OdsCode] = SOURCE.[OdsCode],
+                TARGET.[CrmRef] = SOURCE.[CrmRef],
+                TARGET.[Deleted] = SOURCE.[Deleted],
+                TARGET.[LastUpdated] = SOURCE.[LastUpdated],
+                TARGET.[LastUpdatedBy] = SOURCE.[LastUpdatedBy]
+WHEN NOT MATCHED BY TARGET THEN  
+    INSERT  ([Id], [Name], [LegalName], [Summary], [SupplierUrl], [Address], [OdsCode], [CrmRef], [Deleted], [LastUpdated], [LastUpdatedBy])
+    VALUES  (SOURCE.[Id], SOURCE.[Name], SOURCE.[LegalName], SOURCE.[Summary], SOURCE.[SupplierUrl], SOURCE.[Address], SOURCE.[OdsCode], SOURCE.[CrmRef], SOURCE.[Deleted], SOURCE.[LastUpdated], SOURCE.[LastUpdatedBy]);
+END;
+GO
```

Comment:
As with the solutions, I think it might be better to consolidate the various single-line supplier inserts that are currently spread throughout the file as a single INSERT statement. What do you think?

Reply:
As above, I think I'd rather leave as is.